CheckersClient/src/main/webapp/checkers.jsp

```diff
@@ -1,5 +1,29 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
 <html>
-<body>
-<h2>Hello World!</h2>
-</body>
-</html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+<head><title>Checkers Game - Enter</title></head>
+<style>
+div
+{
+    text-align: center;
+    color: white;
+    font-family: verdana;
+    font-size: 20px;
+}
+</style>
+<body style="background-color:red;">
+<div>
+<br><h3>Welcome to the Checkers Game.</h3><br>
+<br><h3><a href = https://simple.wikipedia.org/wiki/Checkers>How to Play!</a> </h3><br>
```

Comment:
Is it not working? When I tested it, it was working?

checked by source:
Example
The href attribute specifies the link's destination:

<a href="http://www.w3schools.com">Visit W3Schools</a>